lib/resources.rb

```diff
@@ -3237,6 +3237,72 @@ def get_resource(key)
       ]
     }
 
+    MAY2015_MEDICATION_STATEMENT_CREATE ||= {
+      "resourceType": "MedicationStatement",
+      "contained":
+      [
+        {
+          "resourceType":"Medication",
+          "id":"123",
+          "code":{
+            "text":"FHIR Test Medication"
+          }
+        }
+      ],
+      "patient": {
+        "reference":"Patient/234"
+      },
+      "status":"in-progress",
+      "medicationReference":{
+        "reference":"#123"
+      },
+      "dosage":
+      [
+        {
+          "schedule":{
+            "code":{
+              "coding":
+              [
+                {
+                  "system":"http://hl7.org/fhir/v3/vs/GTSAbbreviation",
+                  "code":"BID"
+                }
+              ],
+              "text":"BID"
+            }
+          },
+          "quantity":{
+            "value":60.0,
+            "units":"mg",
+            "system":"http://unitsofmeasure.org",
+            "code":"mg"
+          }
+        }
+      ]
+    }
+
+    MAY2015_MEDICATION_STATEMENT_UPDATE ||= {
+      "resourceType": "MedicationStatement",
+      "id": "222",
+      "contained":
```

Comment:
@jfrhong Based on your comment below about limiting contained to create, should this be removed from the update example?